VRDR/IJEMortality.cs

```diff
@@ -777,8 +777,8 @@ private void Set_MappingIJEToFHIR(Dictionary<string, string> mapping, string ije
         /// <summary>NCHS ICD10 to actual ICD10 </summary>
         private string NCHSICD10toActualICD10(string nchsicd10code)
         {
-            Regex ICD10rgx = new Regex(@"^[A-Z]\d{2}(\.\d){0,1}$");
-            Regex NCHSICD10rgx = new Regex(@"^[A-Z]\d{2,3}$");
+            Regex ICD10rgx = new Regex(@"^[A-Z]\d{2}(\.\d){0,1}$");  // ICD10 codes only have one character after the decimal
```

Comment:
Can't ICD codes can have multiple characters after the decimal? Note https://www.webpt.com/blog/understanding-icd-10-code-structure/

This is mostly relevant for the comments, but might make sense to update the regex to match.

Reply:
Looking at https://www.johndcook.com/blog/2019/05/05/regex_icd_codes/, 
The letter can't be U, so they recommend:
[A-TV-Z][0-9][0-9AB]\.?[0-9A-TV-Z]{0,4}